python3-hello/README.md

```diff
@@ -0,0 +1,407 @@
+
+```console
+test -d ../repos/unikraft || git clone https://github.com/unikraft/unikraft ....//repos/unikraft
+test -d ../repos/libs/musl || git clone https://github.com/unikraft/lib-musl ../repos/libs/musl
+test -d ../repos/libs/lwip || git clone https://github.com/unikraft/lib-lwip ../repos/libs/lwip
+test -d ../repos/libs/python3 || git clone https://github.com/unikraft/lib-python3 ../repos/libs/python3
+test -d ../repos/libs/compiler-rt || git clone https://github.com/unikraft/lib-compiler-rt ../repos/libs/compiler-rt
+make distclean
+> /tmp/defconfig echo 'CONFIG_PLAT_KVM=y
+CONFIG_KVM_VMM_QEMU=y
+CONFIG_ARCH_ARM_64=y
+CONFIG_LIBPYTHON3=y
+CONFIG_LIBPYTHON3_MAIN_FUNCTION=y
+CONFIG_LIBVFSCORE=y
+CONFIG_LIBVFSCORE_AUTOMOUNT_UP=y
+CONFIG_LIBRAMFS=y
+CONFIG_LIBUKCPIO=y
+CONFIG_STACK_SIZE_PAGE_ORDER=10
+CONFIG_LIBPOSIX_ENVIRON=y
+CONFIG_LIBPOSIX_ENVIRON_LIBPARAM=y
+CONFIG_ARM64_ERRATUM_858921=n
+CONFIG_ARM64_ERRATUM_835769=n
+CONFIG_ARM64_ERRATUM_843419=n'
+UK_DEFCONFIG=/tmp/defconfig make defconfig
+make -j $(nproc)
+test -d ./rootfs/ || docker build -o ./rootfs -f Dockerfile .
+test -f initrd.cpio || ../repos/unikraft/support/scripts/mkcpio initrd.cpio ./rootfs/
+qemu-system-aarch64 \
+    -nographic \
+    -machine virt \
+    -m 256 \
+    -cpu max \
+    -append "python3-hello_qemu-arm64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /app/hello.py" \
+    -kernel out/python3-hello_qemu-arm64 \
+    -initrd ./initrd.cpio
+```
+
+Similar to the `x86_64` build, this will configure, build and run Python3 Hello on Unikraft.
+
+Information about every step and about other types of builds is detailed below.
+
+## Set Up
+
+Set up the required repositories.
+Clone them in `../repos/` if not already cloned:
+
+```console
+test -d ../repos/unikraft || git clone https://github.com/unikraft/unikraft ....//repos/unikraft
+test -d ../repos/libs/musl || git clone https://github.com/unikraft/lib-musl ../repos/libs/musl
+test -d ../repos/libs/lwip || git clone https://github.com/unikraft/lib-lwip ../repos/libs/lwip
+test -d ../repos/libs/python3 || git clone https://github.com/unikraft/lib-python3 ../repos/libs/python3
+test -d ../repos/libs/compiler-rt || git clone https://github.com/unikraft/lib-compiler-rt ../repos/libs/compiler-rt
+```
+
+If you want use a custom variant of a repository (e.g. apply your own patch, make modifications), update it accordingly in the `../repos/` directory.
+
+## Clean
+
+While not strictly required, it is safest to clean the previous build artifacts:
+
+```console
+make distclean
+```
+
+## Configure
+
+To configure the kernel, use:
+
+```console
+make menuconfig
+```
+
+In the console menu interface, choose the target architecture (x86_64 or ARMv8 or ARMv7) and platform (Xen or KVM/QEMU or KVM/Firecracker).
+After choosing the target architecture and the platform, go back to the `Architecture Selection` menu entry and update the `Stack size page order` to `10`.
+Then save the configuration.
```

Comment:
also mention the CONFIG_LIBPYTHON3_PYTHONPATH here

Reply:
Not required if the `env.vars` command line option is used.